src/preact-render-spy.js

```diff
@@ -249,6 +296,32 @@ class FindWrapper {
     }
     rerender();
   }
+
+  find(selector) {
+    return new FindWrapper(this.spy, Array.from(this), selector);
```

Comment:
I feel like the second `find()` should only search starting with the children of the first find, but maybe I'm wrong...